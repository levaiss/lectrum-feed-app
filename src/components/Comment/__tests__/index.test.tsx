// Core
import { render } from '@testing-library/react'

// Components
import { Comment } from '../index'

// Helpers
import { comment } from '../../../tests/fake-data'

describe('Comment component', () => {
  test('Comment structure should be received from props value', () => {
    const { container } = render(<Comment comment = { comment } />)
    expect(container).toMatchSnapshot()
  })
})
