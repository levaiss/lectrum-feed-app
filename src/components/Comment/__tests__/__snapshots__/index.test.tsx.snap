// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment component Comment structure should be received from props value 1`] = `
<div>
  <li
    class="commentBody"
  >
    <p>
      John Dou
      <span
        datetime="1615772222"
      >
        53 years ago
      </span>
    </p>
    <p>
      Test comment from John Dou
    </p>
  </li>
</div>
`;
